build.sbt

```diff
@@ -59,14 +59,33 @@ lazy val xmlScala = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     ),
   )
 
+lazy val csv = crossProject(JVMPlatform, JSPlatform, NativePlatform)
```

Comment:
Make sure to add this to the root aggregate. Then MiMa will start failing, so you will need to add a `tlVersionIntroduced := ...`

Reply:
Thank you for the catch!

I guess I'll never learn to not forget the aggregation 🙈